tests/fuzzing.rs

```diff
@@ -60,3 +60,29 @@ fn fuzz_empty_doctype() {
     ));
     assert_eq!(reader.read_event_into(&mut buf).unwrap(), Event::Eof);
 }
+
+#[test]
+fn fuzz_empty_comment() {
```

Comment:
I think, it would be better to place tests to `tests/issues.rs` file. Also, could you check that we actually returns `UnexpectedToken("<!-->")`? That is, explicitly expect all the events that the parser returns:
```rust
assert_eq!(reader.read_event_into(&mut buf), Ok(...));
assert_eq!(reader.read_event_into(&mut buf), Ok(...));
// and so on
```

Reply:
Indeed. Done.